application/models/card.py

```diff
@@ -5,8 +5,8 @@ class Card(db.Model):
     __tablename__ = 'card'
     name = db.Column(db.String(50), primary_key=True)
     noPlayers = db.Column(db.Integer, primary_key=True)
-    age = db.Column(db.Integer, default=1)
-    type = db.Column(db.String(10))
+    age = db.Column(db.Integer, primary_key=True)
```

Comment:
The primary key consists of all three values together. The three together are unique. Table example:

Name    noPlayers
Tavern     4
Tavern     6
Sawmill   4

Here, name is not unique: there are two cards called Tavern. Likewise, noPlayers is not unique, because there are two cards with 4 in that column. But together, (name, noPlayers) is unique, e.g. no card other than the first has values (Tavern, 4).